pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "open-rarity"
-version = "0.2.1-beta"
+version = "0.3.1-beta"
```

Comment:
This was for https://github.com/OpenRarity/open-rarity/pull/39

Reply:
Should be 0.3.0-beta

Reply:
fixed